### PR TITLE
RSWEB-8478: add class for token table dropdowns

### PR DESCRIPTION
**Pull request by brantdarilek**

### CODE REVIEW

File: styleguide/_themes/global/scss/tables.scss [hunk 1]
```diff
@@ -29,6 +29,10 @@ $table-padding: 20px 5px;
   width: 20%;
 }
 
+.productTable-dropdown {
+  color: $black;
```
Comment (WidgetsBurritos) [inline styleguide/_themes/global/scss/tables.scss]: @amberRrucker you cool with black here?

**Comment by amberRrucker** [comment]
+1 black is totally fine

**Comment by WidgetsBurritos** [comment]
Can one of the admins verify this patch?

**Comment by WidgetsBurritos** [comment]
Can one of the admins verify this patch?

**Comment by WidgetsBurritos** [comment]
Can one of the admins verify this patch?

**Pull request merged**